README.md

```diff
@@ -1 +1,19 @@
-# ak-partner-rsvp
\ No newline at end of file
+# ActionKit Partner RSVP Export
+
+This is a tool to allow partners to export their own sourced RSVPs from ActionKit event campaigns. It has two parts: a static site and an API.
```

Comment:
ok, maybe call this a 'request page' rather than a static site -- "static site" for me evokes non-dynamic page content like documentation.  This 'static' page's only purpose is to make a dynamic request.

Reply:
I updated this language a little to make it clearer that the static site calls the API, but I maintain it is a static site by the common meaning. See, for example, https://en.wikipedia.org/wiki/Static_web_page